// Copyright (c) 2015 Cloudera, Inc.
// Confidential Cloudera Information: Covered by NDA.
package kudu;

// Dependency chain:
//
// this file --> proto_container_test.proto

import "kudu/util/proto_container_test.proto";

// Arbitrary protobuf that has one PB dependency.
message ProtoContainerTest2PB {
  required kudu.ProtoContainerTestPB record = 1;
}
